src/simularium/ISimulator.ts

```diff
@@ -18,7 +18,7 @@ export interface ISimulator {
     /**
      * Connect
      * */
-    connectToUri(uri: string): void;
+    createWebSocket(uri: string): void;
```

Comment:
I hope this can be removed from the ISimulator interface and it's really just private to RemoteSimulator.

Reply:
Removed from ISimulator, ClientSimulator, and LocalSimulator and seems like everything still works fine.